raiden-dapp/src/services/raiden-service.ts

```diff
@@ -170,7 +169,6 @@ export default class RaidenService {
         address: tokenAddress
       };
     } catch (e) {
-      console.error(e);
```

Comment:
The deletion of `console.error()` above makes sense since we push an error into the store instead. But here it might make sense to have console output if something goes wrong

Reply:
I would argue that it would be better to enhance the error message with an expandable area (show error message) and show it in the login screen. Instead of logging it in the console.

Reply:
This would be better, but having an error message in the console as having nothing (which is the current state)